src/VCSWebRenderer.ts

```diff
@@ -645,11 +645,23 @@ export default class DailyVCSWebRenderer {
           } else {
             videoEl = document.createElement('video');
             this.placeVideoSourceInDOM(videoEl, video.track.id);
+            videoEl.setAttribute('autoplay', 'true');
+            videoEl.setAttribute('playsinline', 'true');
+            videoEl.setAttribute('controls', 'false');
           }
-          videoEl.srcObject = mediaStream;
-          videoEl.setAttribute('autoplay', 'true');
-          videoEl.setAttribute('playsinline', 'true');
-          videoEl.setAttribute('controls', 'false');
+
+          if (!videoEl.srcObject) {
+            videoEl.srcObject = mediaStream;
+          }
+
+          const srcObject = videoEl.srcObject as MediaStream;
+          const currentVideoTrack = srcObject.getTracks()[0];
```

Comment:
For extra safetiness:
```suggestion
          const currentVideoTrack = srcObject.getVideoTracks()[0];
```